src/main/java/org/oscii/Main.java

```diff
@@ -78,8 +79,14 @@ public static void main(String[] args) throws Exception {
         if (options.has("rank")) {
             ranker = new Ranker((File) options.valueOf("rank"));
         }
-        
-        final LexiconProtocol protocol = new LexiconProtocol(lexicon, corpus, ranker);
+
+        Word2VecManager word2vecMan = null;
```

Comment:
sure, do you want it to be an `EmbeddingsManager` as well? I wasn't very creative to be honest